release/preview/nanoserver1809/docker/Dockerfile

```diff
@@ -0,0 +1,56 @@
+# escape=`
+# Args used by from statements must be defined here:
+ARG fromTag=1709
```

Comment:
How do you make sure `fromTag` and `WindowsServerCoreVersion` are based the same os version (eg, ltsc2019)?

Reply:
They don't need to be.  So I don't

Reply:
`latest` is supposed to be a `manifestlist` that works on the current platform.  So, I used that.  But, after they moved to mcr, it's no longer a `manifestlist`.  I deal with the problem of incompatible kernels if I ever hit it.

Reply:
I see. Thanks.